src/handlers/transformation/build-event-name.ts

```diff
@@ -0,0 +1,34 @@
+import { CsvRow } from "./transform-row";
+
+interface Rules {
+  "Minimum Level Of Assurance": string;
+  "Billable Status": string;
+  "Event Name": string;
```

Comment:
Why are the keys in this interface in Title Case? I hope those are not used for display purposes. Would `minLevelOfAssurance` do?

Reply:
Added a TO DO